nototools/lint_config.txt

```diff
@@ -11,3 +11,23 @@ disable gpos/missing
 filename like Bengali
 enable script_required except cp 951-952
 
+condition
+vendor Adobe
+
+# Adobe's position is that these (mathematical) characters are intended to be used with CJK
+# in an LTR environment.  For true bidi text (that would naturally trigger bidi) these
+# characters should come from a font designed for bidi.  So there is no need in this font
+# for the mirrored glyphs.
+disable bidi
+
+# The v1.003 fonts align the windows ascent/descent with the hhea values, but not the
+# typographic ascender
+version == 1.003
+disable head/os2/ascender, head/os2/descender
+
+# Adobe says "None of the zero-width glyphs are involved with attachment"
```

Comment:
Run the lint tool on the CJK fonts, which were pushed yesterday.  There are hundreds.

For example. NotoSansCJKsc-Thin.otf reported this:
"The following 919 glyphs are unreachable in the font: cid01357,
> cid01676, cid04122, cid06950, cid13116, cid13678, cid61783, cid61784,
> cid61785, cid61788, cid61789, cid61790, cid61791, cid61793, cid61794,
...

Adobe replied:

"Except for a very small number of exceptions (whose glyphs are likely to be exorcized in the Version 2.000 update), all glyphs are reachable by some means. Sometimes, it depends on the font, due to default language. CID+1676, for example, is reachable via the 'hist' GSUB feature. Sometimes, more than one GSUB feature needs to be used, such as 'ccmp' then 'vert'."

We need to investigate this.  There are other issues with the unreachable glyph implementation as well.  It seems reasonable at the moment to disable the test while we investigate and figure out why we're seeing such a difference from what Adobe sees with their tools.